index.html

```diff
@@ -188,7 +213,7 @@ <h3><img src="image/bullets/Rectangle_57bullet.svg" alt="O">Skills</h3>
                 </div>
             </div>
         </section>
-        <section>
+        <section class="contact-section" id="contact">
             <div class="contact">
                 <h3>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h3>
                 <form action="https://formspree.io/f/xayvlole" method="post">
```

Comment:
- [ ] Please, kindly adjust your contact form to be close to the figma design below: 
**FIGMA DESIGN**

![FORM](https://user-images.githubusercontent.com/59397159/165849545-129a2117-94ab-45a3-a6ef-278af9ca7812.png)

**YOUR DESIGN**

![YOUFORM](https://user-images.githubusercontent.com/59397159/165849541-fdb4d0b2-f9a0-4284-87e2-e640a7b5a1cc.png)

- The button is aligned to the left.
- Adjust the width of the textarea and input 
- Reduce the space between the paragraph and the form